ledger-core/src/core/database/DatabaseSessionPool.cpp

```diff
@@ -39,8 +43,8 @@ namespace ledger {
             const std::shared_ptr<api::PathResolver> &resolver,
             const std::shared_ptr<spdlog::logger>& logger,
             const std::string &dbName,
-            const std::string &password
-        ) : _pool((size_t) backend->getConnectionPoolSize()), _backend(backend), _buffer("SQL", logger) {
+            const std::string &password) : 
```

Comment:
Are you kidding me.

Reply:
Alright alright I just copy-pasted changes from PR (even if there are none modifications here).

Let's `clang-format` to that stuff for us.

Reply:
Yeah, we need to introduce it! :)